Confus/Confus/MazeGenerator.cpp

```diff
@@ -160,6 +160,11 @@ namespace Confus
         }
     }
 
+    void MazeGenerator::resetMaze()
+    {
+
+    }
```

Comment:
Deleted this, since there is a chance the only the server knows the initial seed, the server will just send the initial seed again.